src/clients/Wyam/Commands/ConfigOptions.cs

```diff
@@ -13,6 +13,7 @@ internal class ConfigOptions
         public bool UseGlobalSources = false;
         public DirectoryPath PackagesPath = null;
         public bool OutputScript = false;
```

Comment:
We should probably add an extra option (with a corresponding CLI flag) like `OutputConfigAssembly` with a default value of `true` to allow opt-out of caching the assembly if the user wants to.

And if we're not writing the assembly, we probably shouldn't write the hash either (otherwise a previously saved assembly cache and the new hash may be inconsistent).